eos/form-factors/parametric-kkvdz2022.cc

```diff
@@ -48,8 +51,36 @@ namespace eos
                  UsedParameter(p[_par_name("rho_3_2")],  *this) },
                { UsedParameter(p[_par_name("rho_4_0")],  *this),
                  UsedParameter(p[_par_name("rho_4_1")],  *this),
-                 UsedParameter(p[_par_name("rho_4_2")],  *this) }}}
-    {
+                 UsedParameter(p[_par_name("rho_4_2")],  *this) }}},
+        _c_omega{{{ UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this) },
+                 { UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this) },
+                 { UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this) },
+                 { UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this),
+                   UsedParameter(p[0.0],  *this) }}},
+        _c_rho{{{ UsedParameter(p[0.0],  *this),
```

Comment:
Hm. Can you distinguish the subtraction constants in the ``rho`` dispersion relation from those of the ``omega`` dispersion relation?